src/Symfony/Component/Notifier/Bridge/Telnyx/TelnyxTransportFactory.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\Telnyx;
+
+use Symfony\Component\Notifier\Exception\UnsupportedSchemeException;
+use Symfony\Component\Notifier\Transport\AbstractTransportFactory;
+use Symfony\Component\Notifier\Transport\Dsn;
+use Symfony\Component\Notifier\Transport\TransportInterface;
+
+/**
+ * @author Mihail Krasilnikov <mihail.krasilnikov.j@gmail.com>
+ *
+ * @experimental in 5.3
+ */
+final class TelnyxTransportFactory extends AbstractTransportFactory
+{
+    /**
+     * @return TelnyxTransport
+     */
+    public function create(Dsn $dsn): TransportInterface
+    {
+        $scheme = $dsn->getScheme();
+        $apiKey = $dsn->getUser();
+        $host = 'default' === $dsn->getHost() ? null : $dsn->getHost();
+        $from = $dsn->getOption('from');
```

Comment:
```suggestion
        $from = $dsn->getOption('from');
        
        if (!$from) {
            throw new IncompleteDsnException('Missing from.', $dsn->getOriginalDsn());
        }
        
```

It looks like `from` is required

Reply:
Done